tests/TestableMessageHandler.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace SymfonyCasts\MessengerMonitorBundle\Tests;
+
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+final class TestableMessageHandler implements MessageHandlerInterface
+{
+    public function __invoke(TestableMessage $message)
+    {
+        if (true === $message->willFail) {
+            $message->willFail = false;
+            throw new \Exception('oops!');
```

Comment:
when tests suite runs, it displays a log (ex: `[critical] Error thrown while handling message SymfonyCasts\MessengerMonitorBundle\Tests\TestableMessage. Removing from transport after 0 retries. Error: "oops!"`) see the github action...
i don't know how to hide it...